py/test/selenium/webdriver/common/firefox_print_pdf_tests.py

```diff
@@ -0,0 +1,51 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
```

Comment:
Nearly there. 

Change this file to be called `print_pdf_tests.py`

Disable the tests with `pytest.mark.xfail_{insert browser that doesn't work here}`